x-pack/plugins/case/server/services/index.ts

```diff
@@ -45,20 +63,50 @@ interface PushedArgs {
 }
 
 interface GetCaseArgs extends ClientArgs {
-  caseId: string;
+  id: string;
 }
 
 interface GetCasesArgs extends ClientArgs {
   caseIds: string[];
 }
 
-interface FindCommentsArgs extends GetCaseArgs {
+interface GetSubCasesArgs extends ClientArgs {
+  ids: string[];
+}
+
+interface FindCommentsArgs {
+  client: SavedObjectsClientContract;
+  id: string | string[];
+  options?: SavedObjectFindOptions;
+}
+
+interface FindCaseCommentsArgs {
+  client: SavedObjectsClientContract;
+  id: string | string[];
+  options?: SavedObjectFindOptions;
+  includeSubCaseComments?: boolean;
```

Comment:
nit: 
```suggestion
interface FindCaseCommentsArgs extends FindCommentsArgs{
  includeSubCaseComments?: boolean;
```

Reply:
Thanks for these suggestions. Is it ok if I push these to the next refactoring PR? When we split up the services we can refactor these types too.

Reply:
Of course!